### PR TITLE
Add documentation for the probably unknown feature of function multiplication

### PR DESCRIPTION
**Pull request by Lyxal**

### CODE REVIEW

**Review by SE-Nitro — APPROVED** [review]

**Comment by AMiller42** [comment]
> Who knew this stuff existed 🤔

Me, only because I know *way* too much about Vyxal and its quirks.

File: documents/knowledge/elements.yaml [hunk 1]
```diff
@@ -857,12 +859,13 @@
     - '["X"] : "X"'
 
 - element: "T"
-  name: Truthy Indices
-  description: Get indices of truthy elements or triple
+  name: Truthy Indices / Triadify
+  description: Get indices of truthy elements or make the arity of a function 3
```
Comment (pxeger) [inline documents/knowledge/elements.yaml:863]: This now loses the description for numbers, which is normal tripling

Comment (pxeger) [inline documents/knowledge/elements.yaml:863]: This now loses the description for numbers, which is normal tripling

**Review by pxeger — CHANGES_REQUESTED** [review]

File: documents/knowledge/elements.yaml (hunk 1 above)
Comment (cgccuser) [inline documents/knowledge/elements.yaml:863]:
```suggestion
  name: Truthy Indices / Triple / Triadify
  description: Get indices of truthy elements, triple, or make the arity of a function 3
```

**Review by cgccuser — CHANGES_REQUESTED** [review]
Based on pxeger's comment

**Comment by pxeger** [comment]
> Screw that guy

hey >:(

**Pull request merged**